src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -21,12 +21,9 @@
     <DocumentationFile>bin\Release\net45\MongoDB.Driver.Core.xml</DocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="DnsClient" Version="1.0.7" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1" />
```

Comment:
I'm fine with this, but they were separate on purpose.

The first set of package references configured code analysis.

The second set of packages references configured the APIs the consumed.

I'm sure it's better now since the GUI may act in a non-deterministic fashion when confronted with two sections.

Reply:
Ah, thanks for the explanation Robert! Honestly, updating packages via the GUI is sometimes so painful it's easier to do it by hand!